### DISCUSSION TITLE
SDLC Member Feedback: PR to resolve #1720 Security queries are not implemented to spec

### DISCUSSION THREAD

**Discussion by theresalech** [smartdevicelink/sdl_android#1728] [General]
The SDLC review period for PR #1724 starts now and continues through September 21, 2021. Following the review period, the Project Maintainer will conduct final testing to prepare the PR for release in Java Suite 5.3.

Please leave any any questions, comments, or other feedback about PR #1724 and/or Issue #1720 here, so that necessary discussion can take place as soon as possible.

Thank you!

**Comment by mrapitis** [comment]
We have integrated PR #1724 into a sample app and have performed testing, results have been positive thus far.

**Comment by theresalech** [comment]
The SDLC review period has concluded for this PR. It has been merged and will be released in Java Suite 5.3 on October 27, 2021. Thank you for testing!
